apps/admin/backend/test/setup_custom_matchers.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  ToMatchPdfSnapshotOptions,
+  toMatchPdfSnapshot,
+} from '@votingworks/image-utils';
+import { toMatchImageSnapshot } from 'jest-image-snapshot';
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace jest {
+    interface Matchers<R> {
+      toMatchPdfSnapshot(options?: ToMatchPdfSnapshotOptions): Promise<R>;
+    }
+  }
+}
+
+expect.extend({ toMatchImageSnapshot, toMatchPdfSnapshot });
```

Comment:
Could make it simpler by making `jest-image-snapshot` a real rather than dev dependency of `image-utils`.

I'm hesitant to do stuff like that because I still hope that someday we can install only prod packages on prod machines, and keeping things in dev dependencies would make that easier. But to be fair, we're not close to doing that.